unlock-app/src/components/interface/LockTypeLandingPage.tsx

```diff
@@ -177,12 +177,12 @@ export const LockTypeLandingPage = ({
         {callToAction?.subtitle && (
           <p className="mt-4 text-center md:w-1/2">{callToAction?.subtitle}</p>
         )}
-        <ul className="grid md:grid-cols-[300px_300px_300px] gap-8 my-8">
+        <ul className="grid lg:grid-cols-2 gap-16 my-8 md:w-4/5">
```

Comment:
Can you make sure this does not break the pages were there are 3 boxes for example?